core/src/main/groovy/noe/rhel/server/httpd/HttpdBaseOS.groovy

```diff
@@ -98,22 +81,16 @@ class HttpdBaseOS extends Httpd {
 
   void backupConfs() {
     configDirs.each { dir ->
-      def srcDir = getServerRoot() + dir
-      def bkpDir = srcDir + ".bkp"
-
-      Cmd.executeCommand("sudo cp --recursive --preserve --dereference $srcDir $bkpDir", new File("."))
+      File srcDir = new File(getServerRoot(), dir)
```

Comment:
Is it tested? So there is no problem with access rights or so?

Reply:
It is the same as with BaseOS tomcat or RPM testing, you have to run testSuite as root. There are in deed the problems with visibility and permissions as BaseOS httpd files are owned by root. Running our testSuite as root in our OpenStack is possible.

I can do full support in next iteration, but it takes more time to solve all possible problems.

Reply:
I am confused a little bit.

What you want to achieve exactly in this MR, please? Will be anything broken? Will be any changes in current automation necessary? (we don't want that)

Reply:
No changes needed at all. Just copying the same workflow I am used to for execution of BaseOS products.